### DISCUSSION TITLE
GİTHUB PUAN TOPLAMA

### DISCUSSION THREAD

**Discussion by hrhrrcp** [IBAX-io/go-ibax#1059] [Soru Cevap - Türkçe]
Yapıcı konular önerin

Onaylanan her yapıcı konu 200 puan ile ödüllendirilecek ve ödül her ayın 5'inden sonraki bir hafta içinde (IBAX günü) verilecektir.

Teknik araştırma değeri, iş başvurusu değeri, pazar geliştirme beklentileri, akademik keşif değeri ve diğer konular varsa, ibax Vakfı bunu bir ödül görevine dönüştürecek, ayrıca IBAX Puanları da IBXC Coins ile ödüllendirilecektir.

IBAX güvenlik açıklarını bulun

IBAX blok zincirindeki hataları bulun, yeniden üretim sürecini gönderin, IBAX ekibi tarafından doğrulandıktan sonra 1~10.000 IBXC ödülü ve IBAX puanı ödülü alabilirsiniz.

IBAX ekibi tarafından kabul edilen veya projeyle birleştirilen hatalara kod çözümleri göndermek, IBAX puanlarının yanı sıra 10~50.000 IBXC ile ödüllendirilecektir.

IBAX blok zincirinin performansını ve verimliliğini artırabilecek önemli optimizasyon kodu bulursanız, 10~1.000.000 IBXC ve IBAX puanı ile ödüllendirileceksiniz.

Ödül Sorunlarını Çözün

IBAX ödül alanına gidin, ödül sorunlarını çözün ve IBXC jetonları ve IBAX puan ödülleri kazanın.

Bunlar yazılmış fakat hata bulup kod gönderebilseydik keşke :) Ancak tartışmada yorum yapıp geçeriz. Yorumlar ile de oluşmuyor tabi puanlar.

**Comment by Trajedi032** [comment]
Ödüller bölümünü bulamadım yardımcı olur musunuz?

**Comment by Omertastan2036** [comment]
Bize yaramaz

**Comment by MstfYtskn** [comment]
Nasıl puan arttırabiliriz
